src/093 - CDMyNetwork/C-DMyNetwork/C-DMyNetwork.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!--
+ SPDX-FileCopyrightText: 2009-2020 TRUMPF Laser GmbH, authors: C-Labs
+ SPDX-License-Identifier: MPL-2.0
+ -->
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net45;net40;net35</TargetFrameworks>
+        <Version>4.302.0</Version>
+        <Authors>Chris Muench</Authors>
+        <Company>C-Labs</Company>
+        <Product>C-DMyNetwork</Product>
+        <Description>Network Services</Description>
+        <Copyright>(C) 2017-2020 C-Labs</Copyright>
+        <ProjectName>$(AssemblyName).$(TargetFramework)</ProjectName>
```

Comment:
Should generate a nuget package for this

Reply:
what is missing?

Reply:
<GeneratePackageOnBuild>true</GeneratePackageOnBuild>